test/test.js

```diff
@@ -597,78 +588,20 @@ var topDescribe = function (type, createServer) {
   if (parseInt(process.version.split('.')[0].replace(/^v/, ''), 10) > 11) {
     describe('req.socket', function () {
       it('should not throw when socket is null', function (done) {
-        wrapper(request(createServer(function (req, res, next) {
+        request(createServer(function (req, res, next) {
           res.statusCode = 200
           res.end('ok')
           process.nextTick(function () {
             req.socket = null
             next(new Error())
           })
         }))
-          .get('/'))
+          .get('/')
           .expect(200)
           .end(function (err) {
             done(err)
           })
       })
     })
   }
-
-  describe('no deprecation warnings', function () {
-    it('should respond 404 on no error', function (done) {
-      var warned = false
-
-      process.once('warning', function (warning) {
-        if (/The http2 module is an experimental API/.test(warning)) return
-        assert.fail(warning)
-      })
-
-      wrapper(request(createServer())
-        .head('/foo'))
-        .expect(404)
-        .end(function (err) {
-          assert.strictEqual(warned, false)
-          done(err)
-        })
-    })
-
-    it('should respond 500 on error', function (done) {
-      var warned = false
-
-      process.once('warning', function (warning) {
-        if (/The http2 module is an experimental API/.test(warning)) return
-        assert.fail(warning)
-      })
-
-      var err = createError()
-
-      wrapper(request(createServer(function (req, res) {
-        var done = finalhandler(req, res)
-
-        if (typeof err === 'function') {
-          err(req, res, done)
-          return
-        }
-
-        done(err)
-      }))
-        .head('/foo'))
-        .expect(500)
-        .end(function (err, res) {
-          assert.strictEqual(warned, false)
-          done(err)
-        })
-    })
-  })
```

Comment:
These tests can be removed, as using `http2` no longer emits an experimental warning in any of our supported Node.js versions.